gbra/util/ei_graph.py

```diff
@@ -234,7 +240,7 @@ def load(filename):
             possible_ratings = sorted(list(ratings_set))
 
         # Setup the graph with the range of possible ratings.
-        graph.rating_range = (possible_ratings[0], possible_ratings[1])
+        graph.rating_range = (1, 5)
```

Comment:
why did you change this line, looked correct before?

Reply:
https://github.com/alanefl/graph-based-recommender-attacks/pull/17#pullrequestreview-182894696